Automattic/jetpack
# Search: add notice box component for Record Meter

This PR builds upon https://github.com/Automattic/jetpack/pull/23221 to add the notice box component to record meter. 

#### Changes proposed in this Pull Request:

* Adds a notice box component _(notice-box.jsx)_ to display notices to the user. 

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:

Visit `/wp-admin/admin.php?page=jetpack-search&features=record-meter` and check the notice box is displaying under the chart. The current notice should say `You recently surpassed 100 records and will be automatically upgraded to the next billing tier of 1000 max records`

![image](https://user-images.githubusercontent.com/30754158/156657886-a7d0205f-f0f0-4692-9600-5f31cfd119cd.png)

If you change the record counts and tier limits in `lib/create-data.js` the other notice messages available are as follows:

* If it's unable to locate valid indexed data: 	`We weren't able to properly locate your content for Search` _(displayed in a red urgent notice box)_
* If site has not yet been indexed (eg. does not have a last indexed date): `Your content has not yet been indexed for Search`
* If all is in order, but there are no indexable content items: `We weren't able to locate any content to Search to index. Perhaps you don't yet have any posts or pages?`
* If number of indexed records exceeds the current plan limit: `You recently surpassed xx records and will be automatically upgraded to the next billing tier of xx max records`
* If number of indexed records exceeds 80% of the current plan limit: `You're close to the max amount of records for this billing tier. Once you hit xx indexed records, you'll automatically be billed in the next tier`
